edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/AmqpExceptionsHelper.cs

```diff
@@ -29,6 +32,13 @@ public static AmqpException GetAmqpException(Exception ex)
             return new AmqpException(amqpError);
         }
 
+        static Error GenerateAmqpError(EdgeHubAmqpException exception) => new Error
```

Comment:
This PR has a lot of changes where methods are just being moved around. I thought we agreed we don't need to do that alphabetical ordering.. If so, can you please revert those changes? it is making it very large and hard to review (too many changes). 
Also, it might be better to make wholesale changes like changing the copyright in a separate PR to keep it simple. 
This PR currently has >600 files, which is impossible to review.